docs/antrea-network-policy.md

```diff
@@ -500,9 +513,14 @@ the policies that may affect their workloads.
 - There is a soft limit of 20 on the maximum number of Tier resources that are
   supported. But for optimal performance, it is recommended that the number of
   Tiers in a cluster be less than or equal to 10.
-- The v1alpha1 Policy CRDs support up to 10000 unique priority at policy level.
-  In order to reduce the churn in the agent, it is recommended to set the
+- In order to reduce the churn in the agent, it is recommended to set the policy
   priority within the range 1.0 to 100.0.
+- The v1alpha1 Policy CRDs support up to 10,000 unique priorities at policy level,
+  and up to 50,000 unique priorities at rule level, across all tiers except for
+  the "baseline" tier. For any two policy rules, their rule level priorities are only
+  considered equal if they share the same tier, and have the same policy priority
+  as well as rule priority.
```

Comment:
How to make two policy rules same rule priority? I think it cannot be determined by user? If two policy rule have same tier and same policy priority, they should already be equal?

Reply:
They are only considered equal priority if they are at the same position in its ingress/egress rules. Hence the number of unique priorities in a specific tier-policy priority is determined by the max number of ingress or egress rules the user creates in that tier-policy priority.